Makefile

```diff
@@ -37,7 +37,10 @@ simple-binary:
 	pyinstaller calicoctl/calicoctl.py -ayF --clean
 
 calico_node/.calico_node.created: $(NODE_CONTAINER_FILES)
-	cd calico_node && docker build -t calico/node:latest .
+	cd calico_node && docker build -t calico/node:latest \
+		--build-arg VCS_URL=`git config --get remote.origin.url` \
```

Comment:
This is a tad nitpicky, but my personal git checkout doesn't have a `origin` - I've renamed it to `calico` to prevent myself from pushing a branch directly instead of pushing to my fork and opening a PR. 

So when I run this `git config` line, it returns blank. Could we just hardcode this to be `https://github.com/projectcalico/calico-containers.git`?